src/DotNetWorker/RpcExtensions.cs

```diff
@@ -187,9 +187,9 @@ internal static TypedData ToRpcLongArray(this long[] arrLong)
 
             return new RpcException
             {
-                Message = exception.Message,
-                Source = exception.Source,
-                StackTrace = exception.StackTrace
+                Message = $"{exception.GetType()}: {exception.Message}",
```

Comment:
There was no way to tell the type of the exception... so I'm formatting the message like this to include those details. Maybe I should just do `exception.ToString()` here?

Reply:
Yeah, I think that would be preferable (actually, just exception)